README.md

```diff
@@ -3,6 +3,8 @@ About qutip
 
 Home: http://qutip.org
 
+GitHub: https://github.com/qutip/qutip
```

Comment:
Thanks @ajgpitch, but this will be nuked once the feedstock is regenerated. We can certainly consider having this kind of capability though - ideally this would live in the conda recipe metadata.